bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeBand.scala

```diff
@@ -2,20 +2,16 @@ package edu.gemini.spModel.core
 
 import edu.gemini.spModel.core.WavelengthConversions._
 
-sealed abstract class MagnitudeBand private (val name: String, val center: Wavelength, val width: Wavelength, val description: Option[String], val defaultSystem: MagnitudeSystem) extends Product with Serializable {
+sealed abstract class MagnitudeBand private (val name: String, val start: Wavelength, val center: Wavelength, val end: Wavelength, val description: Option[String], val defaultSystem: MagnitudeSystem) extends Product with Serializable {
```

Comment:
Why this change? Do we need to rely on a triplet `start`, `center`, `end` rather than just `center` and `width`?

I guess we'd need to replicate this on gpp